chartpress.py

```diff
@@ -537,7 +536,7 @@ def build_chart(name, version=None, paths=None, long=False):
     return version
 
 
-def publish_pages(chart_name, chart_version, chart_repo_github_path, chart_repo_url, extra_message=''):
+def publish_pages(chart_name, chart_version, chart_repo_github_path, chart_repo_url, extra_message, force):
```

Comment:
```suggestion
def publish_pages(chart_name, chart_version, chart_repo_github_path, chart_repo_url, extra_message='', force=False):
```
would keep the function signature the same. I don't know who/what code relies on `extra_message` being an optional keyword argument. Instead of finding out it would be easier to not change the signature.

Reply:
This function is always called with all its arguments passed explicitly with defaults from the args parser, I felt by not providing another set of defaults here we would avoid confusion of having a duplicated sets of defaults where only one is actually used.

This would be a breaking change of chartpress as a Python library though where this function is called directly, but I don't know anyone doing that and we have made several library API changes before.

I can switch back to providing defaults, but I think we are inconsistent with this and it will cause a bit of confusion, I know for me it has been a bit confusing to have defaults that wasn't used and sometimes have been out of sync with a CLI like this.

Reply:
I added it back as it is a trivial un-important change.